packages/clerk-js/src/ui/hooks/useFetch.ts

```diff
@@ -35,7 +35,7 @@ export const useFetch = <T>(
         requestStatus.setError();
         setData(null);
       });
-  }, []);
+  }, [JSON.stringify(params)]);
```

Comment:
❓ Can you elaborate on this change? Is it necessary to be part of this PR?

Reply:
Previously useFetch would only fire once, "on mount", there was a bug where the Gate component would not hide the content while not having the necessary permission because the "fetch" wouldn't fire when the parameters changed.

For context we are using `useFetch` to handle the async logic of isAuthorized.

Reply:
This would only invoke a call if params change, not on every render.